README.md

```diff
@@ -87,23 +87,23 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
 [rl-arm64]: https://github.com/PowerShell/PowerShell/releases/download/v7.0.0/powershell-7.0.0-linux-arm64.tar.gz
```

Comment:
@TravisEz13, your last commit had 1 failures in `PowerShell-CI-linux`
Requires tests.Version checks.Check for version 7.1
```powershell
Expected an exception, with type [System.Management.Automation.ScriptRequiresException] to be thrown, but no exception was thrown.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/Requires.Tests.ps1: line 90
90:             { . $File } | Should -Throw -ExceptionType ([System.Management.Automation.ScriptRequiresException])
```